google/cloud/storage/transfer_manager.py

```diff
@@ -696,6 +709,11 @@ def download_many_to_path(
         and the default is a conservative number that should work okay in most
         cases without consuming excessive resources.
 
+    :type skip_if_exists: bool
+    :param skip_if_exists:
+        Before downloading each blob, check if the file for the filename exists;
+        if it does, skip that blob. This only works for filenames.
```

Comment:
nit: probably can remove last sentence since this method only applies to file paths (str)

Reply:
fixed